packages/mantine/src/components/ellipsis-text/EllipsisText.tsx

```diff
@@ -52,7 +52,7 @@ export const EllipsisText = polymorphicFactory<EllipsisTextFactory>((props, ref)
     });
 
     const [showTooltip, setShowTooltip] = useState(false);
-    const textRef = useRef<HTMLDivElement>();
+    const textRef = useRef<HTMLDivElement>(null);
```

Comment:
EllipsisText is broken 
![image](https://github.com/user-attachments/assets/283b93be-dbc1-4de7-911a-6eb8623ed39b)